google-cloud-tasks/docs/toc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "guides": [],
+  "services": [
+    {
+      "title": "Tasks",
+      "type": "google/cloud/tasks",
+      "patterns": ["\/tasks", "google\/api", "google\/protobuf"],
+      "nav": [
+        {
+          "title": "V2beta2",
+          "type": "google/cloud/tasks/v2beta2",
+          "patterns": ["\/tasks\/v2beta2", "google\/api", "google\/protobuf"],
+          "nav": [
+            {
+              "title": "CloudTasksClient",
+              "type": "google/cloud/tasks/v2beta2/cloudtasksclient"
+            },
+            {
+              "title": "Data Types",
+              "type": "google/tasks/v2beta2/datatypes"
```

Comment:
I believe this `type` value is incorrect. The correct value is `google/cloud/tasks/v2beta2/datatypes`. (The jsondoc output file is `google-cloud-tasks/jsondoc/google/cloud/tasks/v2beta2/datatypes.json`, derived from the `title` in the `toc_config` in the `Rakefile`: `title: "Google::Cloud::Tasks::V2beta2::DataTypes"`.

Reply:
Is there any way to be sure of this file? This, as best as I can tell, is always handcrafted?

Reply:
I'm not sure I understand your question, but the `type` paths should correspond to files in the `jsondoc` directory (after running `rake jsondoc`).

Reply:
@crwilcox yes, this is something that is always added manually. The best way to be sure is to look through the docs to see if it's being rendered correctly. Once it's merged to master it's easy to simply check the doc site. Prior to that you can preview it locally, but it's a little more work. @quartzmo sent me some good instruction for doing it a few weeks ago.  I can probably dig up for you if you think it would help - I've only done it a few times myself.